Makefile

```diff
@@ -0,0 +1,86 @@
+.DEFAULT_GOAL := help
+
+PSYSH_SRC = bin src box.json.dist composer.json composer.lock build/stub
+
+.PHONY: help
+help:
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
+
+
+##
+## Commands
+##---------------------------------------------------------------------------
+
+clean: 	 ## Clean all created artifacts
+.PHONY: clean
+clean:
+	rm -rf build/
```

Comment:
so I updated the script, there is only two artefact directories ever created: `dist` and `build`. Everything is in there